### DISCUSSION TITLE
Items stuck in "pending uploads" create duplicates on OSM (v4.0, November 2023)

### DISCUSSION THREAD

**Discussion by sekundemal** [Zverik/every_door#676] [General]
I am using Every Door 4.0 on Android (Pixel 6a) and was recently made aware by an OSM user that I had created multiple duplicates of the same map changes. I then went to check the "pending uploads" on the app and found about 90 already uploaded items that would every single time I pressed upload. The only solution was to manually delete these items from pending uploads. I know this problem/bug was discussed here before. It doesn't look like it was fixed. See: http://osmose.openstreetmap.fr/fr/issues/open?country=namibia&item=xxxx&username=Sekundemal

**Comment by mnalis** [comment]
Please do not delete them for the moment, they might be useful to troubleshoot that elusive issue. 

Can you do `(menu)` / `About Every Door` / `System log` / `(send icon)` after doing the upload which duplicates nodes?

- ref: https://github.com/Zverik/every_door/issues/605

**Reply by sekundemal** [reply]
I have sent the system log through the app. Here's a copy. I see that my app has in the meantime updated to version 4.1. The issue has not reoccurred since I deleted the 90 items and cleared the cache. But I will monitor this closely.

[19:13:03] Flutter: PlatformException(error, info.zverev.ilya.every_door: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts, null, null)
[19:13:03] #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:652)
#1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:310)
<asynchronous suspension>
#2      EventChannel.receiveBroadcastStream.<anonymous closure> (package:flutter/src/services/platform_channel.dart:652)
<asynchronous suspension>

**Reply by mnalis** [reply]
Hmmm, it seems less informative that one would've hoped...

If it does occur again, please (along with copy/pasting a log as you did here) also share an `.osc` file as described here: https://github.com/Zverik/every_door/issues/605#issuecomment-1808691767, and (if you can) refrain from clearing the queue for a little while, so people can try to investigate/reproduce.

Thanks again for reporting the issue!

**Reply by sekundemal** [reply]
Thanks, I will post again one the bug occurs again (now for a change hoping that it will).
